bin/cml/repository.js

```diff
@@ -0,0 +1,8 @@
+exports.command = 'repository';
```

Comment:
How does `ci` describe the noun that users are interacting with? That command is all about performing Git operations.

Reply:
Still thinking that the repository concept is better, although as per https://primer.style/cli/foundations/language, `cml repo` is better than `cml repository`

Reply:
Then, `cml ci fix` it is?

Reply:
Taking the Git committer identity configuration as a CI fix is also a bit extreme... _from an engineering perspective_.

Reply:
@casperdcl & @iterative/cml, how should we name this?

Reply:
I think that keeping it as `cml ci` is fine, so is `cml ci fix`. I don't have strong feelings but I am against this being `cml repo/repository` as I feel that has a far different connotation.

Reply:
`cml ci [setup]`?

Reply:
Note that `cml ci` is already a legacy command that should keep working without arguments. If we overload the old name, running `cml ci` won't show the help menu, but rather “so something” users might not expect.

Reply:
Omitting this insignificant technical detail, I'm inclined to think that, given that this command modifies the cloned Git repository configuration, `repo` would be a more accurate noun.

Reply:
Anyway. 🤷🏼‍♂️ Not wanting to make a blocker out of this.

Reply:
Im here with @casperdcl `ci` sounds to me better as its solving a CI case

Reply:
From a purist standpoint, the legacy `cml ci` command was almost a shorthand for a fictitious `git config --auto` and, as such, isn't inherently related to continuous integration.

Reply:
Retaining compatibility for both `cml ci` and `cml ci {verb}` is not possible without sacrificing the help menu. If choosing a name other than `ci` is not possible, we'll have to get very cretive.

Reply:
@iterative/cml, this is the only unresolved conversation, let's do something about it.

Reply:
final thought here: because future functionality isn't clear (e.g. `cml aws docker-setup` or `cml config dvc`) let's just leave `cml ci` as-is for now. No need to do `cml ci {verb}` nor `cml repo...`.

Reply:
Excerpt of my chain of thoughts for the next time we have to take a look at this:

> When this command was introduced we already had some conversations about the description in [cml.dev](http://cml.dev/) because its purpose was rather ethereal.
> 
> * What we say it does: fix the CI system, holistically
> * What we think it does: fix the misdeeds made by different CI/CD systems so repositories behave the same on each of them
> * What it really does: change some configuration options in the cloned repository with git config — shorthand to customize committer name & email, remote authentication, etc.
> 
> Rather cml git config[ure] (noun/verb) but it would be a rather misleading name. :thinking_face:
> 
> I’d stay with `repo config/setup` or similar, but I’d fall into my own trap: repo is also overloaded, meaning both the cloned repository and the remote repository [settings] :exploding_head:

Reply:
For now, it will remain as `cml ci` without any “modern” counterpart.

Reply:
Resolved with https://github.com/iterative/cml/pull/1073/commits/f01975ab39b0dca904f636bb8edc9d1aac37c3e1; internal naming & structure follows the command documentation at https://cml.dev/doc/ref/ci for the sake of sanity.